libp2p/Host.cpp

```diff
@@ -989,13 +997,35 @@ bool Host::peerSlotsAvailable(Host::PeerSlotType _type /*= Ingress*/)
     return peerCount() + m_pendingPeerConns.size() < peerSlots(_type);
 }
 
-KeyPair Host::networkAlias(bytesConstRef _b)
+std::pair<Secret, ENR> Host::restoreENR(bytesConstRef _b, NetworkConfig const& _netConfig)
 {
     RLP r(_b);
+    Secret secret;
     if (r.itemCount() == 3 && r[0].isInt() && r[0].toInt<unsigned>() >= 3)
-        return KeyPair(Secret(r[1].toBytes()));
+    {
+        if (r[1].isList())
+        {
+            secret = Secret{r[1][0].toBytes()};
+            auto enrRlp = r[1][1];
+
+            return make_pair(secret, parseV4ENR(enrRlp));
```

Comment:
std:: prefix (for make_pair)?

Reply:
This file has `using namespace std`, so I follow surrounding code's convention

Reply:
Okay, I called this out since I saw some use of std:: in the file...but if the std:: namespace is being imported then ultimately it doesn't matter.